cgmanifest.json

```
@@ -22440,6 +22440,16 @@
         }
       }
     },
+    {
+      "component": {
+        "type": "other",
+        "other": {
+          "name": "rpm-mpi-hooks",
+          "version": "6",
+          "downloadUrl": "https://src.fedoraproject.org/rpms/rpm-mpi-hooks/blob/rawhide/f/mpi.attr"
```

For this one we'd need to use this:
```suggestion
          "downloadUrl": "https://src.fedoraproject.org/rpms/rpm-mpi-hooks/raw/rawhide/f/mpi.attr"
```

We want only the file, not the whole web page.